<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>LoadDmpDlg</class>
  <widget class="QWidget" name="LoadDmpDlg" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>343</width>
        <height>272</height>
      </rect>
    </property>
    <layout class="QVBoxLayout" >
      <item>
        <layout class="QGridLayout" >
          <item row="2" column="0" >
            <widget class="QLabel" name="textLabel5" >
              <property name="text" >
                <string>Load into folder:</string>
              </property>
              <property name="alignment" >
                <set>Qt::AlignVCenter|Qt::AlignRight</set>
              </property>
              <property name="toolTip" stdset="0" >
                <string>Path to load the dump into (see contexthelp)</string>
              </property>
              <property name="whatsThis" stdset="0" >
                <string>If not empty, load the dump into a specific folder instead into root of repository. This folder must exist before loading the dump.</string>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="0" column="0" >
            <widget class="QLabel" name="textLabel3" >
              <property name="text" >
                <string>Dump file:</string>
              </property>
              <property name="alignment" >
                <set>Qt::AlignVCenter|Qt::AlignRight</set>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="0" column="1" >
            <widget class="KUrlRequester" name="m_Dumpfile" />
          </item>
          <item row="1" column="0" >
            <widget class="QLabel" name="textLabel4" >
              <property name="text" >
                <string>Load into repository:</string>
              </property>
              <property name="alignment" >
                <set>Qt::AlignVCenter|Qt::AlignRight</set>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="2" column="1" >
            <widget class="KLineEdit" name="m_Rootfolder" >
              <property name="toolTip" stdset="0" >
                <string>Path to load the dump into (see contexthelp)</string>
              </property>
              <property name="whatsThis" stdset="0" >
                <string>If not empty, load the dump into a specific folder instead into root of repository. This folder must exist before loading the dump.</string>
              </property>
            </widget>
          </item>
          <item row="1" column="1" >
            <widget class="KUrlRequester" name="m_Repository" >
              <property name="mode" >
                <number>18</number>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <widget class="Q3ButtonGroup" name="m_UuidGroup" >
          <property name="title" >
            <string>Uuid action</string>
          </property>
          <property name="toolTip" stdset="0" >
            <string>How to handle UUIDs</string>
          </property>
          <property name="whatsThis" stdset="0" >
            <string>The repository's UUID will be updated if the dumpstream contains a UUID and action isn't set to ignore and either the repository contains no revisions or action is set to force. If the dump contains no UUID than this action is ignored.</string>
          </property>
          <layout class="QVBoxLayout" >
            <item>
              <widget class="QRadioButton" name="m_UUidDefault" >
                <property name="text" >
                  <string>Default</string>
                </property>
                <property name="shortcut" >
                  <string/>
                </property>
                <property name="checked" >
                  <bool>true</bool>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QRadioButton" name="m_UUidIgnore" >
                <property name="text" >
                  <string>Ignore</string>
                </property>
                <property name="shortcut" >
                  <string/>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QRadioButton" name="m_UUidForce" >
                <property name="text" >
                  <string>Force</string>
                </property>
                <property name="shortcut" >
                  <string/>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="QCheckBox" name="m_UsePre" >
          <property name="text" >
            <string>Use pre-commit hook</string>
          </property>
          <property name="shortcut" >
            <string/>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QCheckBox" name="m_UsePost" >
          <property name="text" >
            <string>Use post-commit hook</string>
          </property>
          <property name="shortcut" >
            <string/>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <includes>
    <include location="local" >kurlrequester.h</include>
    <include location="local" >klineedit.h</include>
    <include location="local" >kpushbutton.h</include>
    <include location="local" >klineedit.h</include>
    <include location="local" >kurlrequester.h</include>
    <include location="local" >klineedit.h</include>
    <include location="local" >kpushbutton.h</include>
  </includes>
</ui>
